app/Http/Controllers/PDFController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use Barryvdh\DomPDF\Facade\Pdf;
+
+class PDFController extends Controller
+{
+
+    public function pdf_generator_get(Request $request)
```

Comment:
You're not using $request, consider removing this.

Reply:
I've removed it